Gruntfile.js

```diff
@@ -15,23 +15,49 @@ module.exports = function(grunt) {
         serverToken: '<%= secrets.serverToken %>'
       },
       email: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Yo',
         src: ['test/email.html']
       },
       bulk: {
-        from: 'you@youremail.com',
-        to: 'you@youremail.com',
+        from: '<%= secrets.testSender %>',
+        to: '<%= secrets.testRecipient %>',
         subject: 'Hey',
         src: ['test/*.html']
       }
-    }
+    },
+
+    "postmark-templates": {
+      options: {
+        serverToken: '<%= secrets.serverToken %>',
+      },
+      build: {
+        name: "testing-template-node-js-" + new Date().valueOf(),
+        subject: "Testing grunt-postmark-templates",
+        // NOTE these are read from filesystem. globbing not supported
+        htmlFile: 'test/email.html',
```

Comment:
After working on this some more, I now agree with you @derekrushforth that these should be `htmlBody` and `textBody`. The entire workflow with mailmason is file based, so it doesn't really make sense to allow literal strings for template bodies.